tests/test_api.py

```diff
@@ -881,6 +881,7 @@ def test_visualize_attributes(self, client, data_model_jsonld):
 
     @pytest.mark.parametrize("figure_type", ["component", "dependency"])
     def test_visualize_tangled_tree_layers(self, client, figure_type, data_model_jsonld):
+        # TODO: Determine a 2nd data model to use for this test, test both models sequentially, add checks for content of response
```

Comment:
I think maybe we should open a Jira issue for this. I don't think we would remember to look at comments